doc/classes/@GlobalScope.xml

```diff
@@ -2143,49 +2143,49 @@
 			Space key.
 		</constant>
 		<constant name="KEY_EXCLAM" value="33" enum="Key">
-			! key.
+			Exclamation mark ([code]![/code]) key.
```

Comment:
In US English this is commonly known as an "_Exclamation point_". however, for consistency with the surrounding descriptions, I'd rather personally call it "_Exclamation mark_". Feel free to argue against this.

Reply:
Both seem perfectly fine to me as a native American English speaker. I don't think the American English standard that Godot uses needs to apply to the choice of terms to use, it's mostly important to standardize spelling

Reply:
I'm from the US and I've only ever heard it referred to as "Exclamation mark".

Reply:
Midwest perspective: I've most commonly heard either "Exclamation Point" or simply "Exclamation;" while comparatively uncommon, it's obvious what "Exclamation Mark" is referring to & it *does* fit better overall